x/crisis/abci_app.go

```diff
@@ -0,0 +1,17 @@
+package crisis
+
+import (
+	"github.com/tendermint/tendermint/libs/log"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// check all registered invariants
+func EndBlocker(ctx sdk.Context, k Keeper, logger log.Logger) {
+
+	if k.invCheckPeriod == 0 || ctx.BlockHeight()%int64(k.invCheckPeriod) != 0 {
```

Comment:
We removed this logic from the gaia app right?

Reply:
correct it was moved to the crisis module (here)